vector/src/main/res/values-fr/strings.xml

```diff
@@ -1588,63 +1082,40 @@
     <string name="login_registration_not_supported">L’application ne peut pas créer de compte sur ce serveur d’accueil.
 \n
-\nVoulez-vous vous inscrire en utilisant un client web \?</string>
-<<<<<<< HEAD
-    <string name="login_login_with_email_error">Cet e\u2011mail n’est associé à aucun compte.</string>
-    <string name="login_reset_password_on">Réinitialiser le mot de passe sur %1$s</string>
-    <string name="login_reset_password_notice">Pour réinitialiser votre mot de passe, saisissez l’adresse e\u2011mail associée à votre compte :</string>
-    <string name="login_reset_password_submit">Suivant</string>
-    <string name="login_reset_password_email_hint">E\u2011mail</string>
-=======
+\nVoulez-vous vous inscrire en utilisant un client web \?</string>
     <string name="login_login_with_email_error">Ce courriel n’est associé à aucun compte.</string>
     <string name="login_reset_password_on">Réinitialiser le mot de passe sur %1$s</string>
     <string name="login_reset_password_notice">Un courriel de vérification sera envoyé à votre adresse pour confirmer la configuration de votre nouveau mot de passe.</string>
     <string name="login_reset_password_submit">Suivant</string>
     <string name="login_reset_password_email_hint">Courriel</string>
->>>>>>> v1.4.8
     <string name="login_reset_password_password_hint">Nouveau mot de passe</string>
-    <string name="login_reset_password_warning_title">Attention !</string>
+    <string name="login_reset_password_warning_title">Attention !</string>
     <string name="login_reset_password_warning_content">Le changement de mot de passe réinitialisera toutes les clés de chiffrement sur toutes vos sessions, rendant l’historique des discussions chiffrées illisible. Configurez la sauvegarde de clés ou exportez vos clés de salon depuis une autre session avant de réinitialiser votre mot de passe.</string>
     <string name="login_reset_password_warning_submit">Poursuivre</string>
-<<<<<<< HEAD
-    <string name="login_reset_password_error_not_found">Cet e\u2011mail n’est lié à aucun compte</string>
-    <string name="login_reset_password_mail_confirmation_title">Vérifiez votre boîte de réception</string>
-    <string name="template_login_reset_password_mail_confirmation_notice">Si un compte ${app_name} existe, un e\u2011mail a été envoyé à l’adresse : %s.</string>
-    <string name="login_reset_password_mail_confirmation_notice_2">Une fois que vous aurez suivi le lien qu’il contient, cliquez ci-dessous.</string>
-    <string name="login_reset_password_mail_confirmation_submit">J’ai vérifié mon adresse e\u2011mail</string>
-=======
     <string name="login_reset_password_error_not_found">Ce courriel n’est lié à aucun compte</string>
     <string name="login_reset_password_mail_confirmation_title">Vérifiez votre boîte de réception</string>
     <string name="login_reset_password_mail_confirmation_notice">Un courriel de vérification a été envoyé à %1$s.</string>
```

Comment:
please restore the tchap translation (the beggining "si un compte Tchap existe"): `Si un compte ${app_name} existe, un courriel de vérification a été envoyé à %1$s.`